google/cloud/forseti/services/model/importer/importer.py

```diff
@@ -92,7 +92,7 @@ def __init__(self,
 
         Args:
             session (object): Database session.
-            model (str): Model name to create.
+            model (Model): Model name to create.
```

Comment:
Do you still need the `name` in the arg description, if this is not `str` type anymore.?

Reply:
Updated the docstring.